packages/firestore/test/integration/api/query.test.ts

```diff
@@ -1525,14 +1817,6 @@ apiDescribe('Queries', (persistence: boolean) => {
       };
 
       return withTestCollection(persistence, testDocs, async coll => {
-        // a==2 || b in [2,3]
-        await checkOnlineAndOfflineResultsMatch(
-          query(coll, or(where('a', '==', 2), where('b', 'in', [2, 3]))),
-          'doc3',
-          'doc4',
-          'doc6'
-        );
-
```

Comment:
the name of the test can be updated now:
`can use or queries with not-in`

Reply:
Updated